### ISSUE TITLE
Installation has created write problem for all but superuser of DokuWiki

### ISSUE THREAD

**Issue by Guy270** [turnermm/ckgedit#670]
Only superuser can write and ckgedit works fine for him.

Others with write access visible in ACL have "Display source text" icon in page menu and upon clicking it see a greyed-out editing window with a mention that the page is read-only. Clicking on the Ckg Editor button on upper left of edit window produces an access denied message. 

Initial install of DokuWiki was some months ago and content has been produced with DW editor since. There is no difference between old content and new content produced by superuser with ckgedit regarding write access by other users with now ineffective write access, even with DW editor. 

Only "uncommon" setting of DokuWiki is that @ALL group has no reading access and [sneaky_index] is turned on (meaning site plan is not visible to @ALL).

**Even after uninstall of ckgedit problem remains**. Only superuser can write with regular editor. Others can only see grayed-out edit window.

So it seems write access for all but superuser has been permanently changed. Individual text file (page) permission is unchanged at 0644. I am no php programmer and do not know where to look for this permanent denial of write access to all but superuser.

I probably could reset normal behavior with a clean reinstall of DokuWiki, but fear that installing ckgedit anew would produce the same result. A wysiwyg editor would be so appreciated, as you can well imagine, so I really need a solution.

Thanks.

**Comment by Guy270** [comment]
And I tested in several browsers. All the same. In the regular one, I deletes the cookies, no difference. The others were used for the first time. In all I switched from superuser admin to user group with write permission and superuser admin always works fine.

**Comment by Guy270** [comment]
Just to be sure, today I reverted all permissions in the ACL but for @ALL that had none to read-only and saved these changes, and checked: of course, can't edit. I then reverted the permissions to previous, "user" group should be able to edit; I checked: nope. ACL changes have no effect, **EXCEPT for @ALL!**
Because I then gave @ALL write permission and it registered; I can not only view pages without login in, I can edit them!
I put @ALL back to its default read-only with no problem.

So it's really a DokuWiki issue, but it seems to have been trigered by the first ckgedit installation. Read-only access is "hard-coded" somewhere for all groups but superuser and @ALL.

Only other thing I did that is unusual was replacing a number of image files with lighter versions through FileZilla ftp in a subdirectory of doku/data/media/ the day before I installed ckgedit. I did not attempt editing a document between the two events.

Il will take up the issue with DokuWiki because it's definitively an ACL problem.
